custom_function_template.gs

```diff
@@ -0,0 +1,154 @@
+/**
+* A custom function template for the following functions - 
+*
+* (operation = 0) Get sum of given values 
+* @return The sum of given values 
+*
+* (operation = 1) Get mean of given values 
+* @return The mean of given values
+*
+* (operation = 2) Get variance for the given input values.
+* @return The variance for the given input values.
+*
+* (operation = 3) Get standard deviation for the given input values.
+* @return The standard deviation for the given input values.
+*
+* (operation = 4) Get all the modes for the given input values.
+* @return The modes for the given input values.
+*
+* (operation = 5) Get maximum value for the given values.
+* @return The maximum value for the given values.
+*
+* (operation = 6) Get countIf - count of input values equal to the given value 
+* @return The count of values equal to the first input value.
+*
+* (operation = 7) Get The median for the given input values.
+* @return The median for the given input values.
+*
+* @customfunction
+*/
+function customFunctionTemplate() {
+  if (arguments.length < 2) {
+    throw new Error("Too few arguments");
+  }
+  var operation = arguments[0]; // First argument determines the operation to be performed.
+  var values = Array.prototype.slice.call(arguments).slice(1); // Input values are second argument onwards. 
+  var len = values.length; // Length of input values (on which function is to be performed).
+  
```

Comment:
At max one empty line between statements

Reply:
Thank you, will make the change.